src/countlyInterceptor.js

```diff
@@ -0,0 +1,73 @@
+const {
```

Comment:
Why can't we write this logic in a typescript file?

Reply:
because we have to import it into a js files without compilation step (into ./bin/dev and ./bin/run). That's how this oclif framework is set up. But this file is still type-checked though

Reply:
But the rest of the project is written in Typescript. It even executes everything through the TS pipeleine (`dev` file):

```
require("ts-node").register({ project });
```

Can't we run this file using the same mechanism?